doc/Manuals/LoadBalancing/ig-lb_x-road_external_load_balancer_installation_guide.md

```diff
@@ -0,0 +1,738 @@
+license, visit http://creativecommons.org/licenses/by-sa/3.0/.
+
+## 1. Introduction
+
+### 1.1 Target Audience
+
+The intended audience of this installation guide are the X-Road security server administrators responsible for installing
+and configuring X-Road security servers to use external load balancing. The document is intended for readers with a good
+knowledge of Linux server management, computer networks, database administration, clustered environments and the X-Road
+functioning principles.
+
+### 1.2 References
+
+| Document Id    |  Document                                                                                |
+|:--------------:|:-----------------------------------------------------------------------------------------|
+| \[SS-CLUSTER\] | [Readme: Security server cluster setup with Ansible](../../../ansible/ss_cluster/README.md) |
+| \[IG-SS\] | [X-Road: Security Server Installation Guide](../ig-ss_x-road_v6_security_server_installation_guide.md) |
+| \[UG-SS\] | [X-Road 6 Security Server User Guide](../) |
+
+
+## 2. Overview
+
+This document describes the external load balancing support features implemented by X-Road and the steps necessary to
+configure security servers to run as a cluster where each node has an identical configuration, including their keys and
+certificates. X-Road security server configuration changes are handled by a single master server and one or more slave
+servers.
+
+### 2.1 Goals and assumptions
+
+The primary goal of the load balancing support is, as the name suggests, load balancing, not fault tolerance.
+A clustered environment increases fault tolerance but some X-Road messages can still be lost if a security server node fails.
+
+The implementation does not include a load balancer component. It should be possible to use any external load balancer
+component that supports HTTP-based health checks for the nodes and load balancing at the TCP level (eg. haproxy, nginx,
+AWS ELB or Classic Load Balancing, or a hardware appliance). A health check service is provided for monitoring a node's
+status, this is described in more detail in section [3.4 Health check service configuration](#34-health-check-service-configuration)
+
+The load balancing support is implemented with a few assumptions about the environment that users should be aware of.
+Carefully consider these assumptions before deciding if the supported features are suitable for your needs.
+
+<a name="basic_assumptions"></a>
+__Basic Assumptions about the load balanced environment:__
+* Adding or removing nodes to or from the cluster is infrequent. New nodes need to be added manually and this takes some
+  time.
+* Changes to the configuration files are relatively infrequent and some downtime in ability to propagate the changes can
+  be tolerated. The cluster uses a master-slave model and the configuration master is not replicated.
```

Comment:
I did not fully understand what is the meaning of "configuration master is not replicated", does it explain the rest of the bullet point somehow? I am guessing it means that having a replicated master would somehow remove the downtime, but not seeing how exactly, and reading this leaves me curious.